.swiftlint.yml

```diff
@@ -2,84 +2,18 @@ allow_zero_lintable_files: true
 excluded:
   - Carthage
   - Pods
-  - MapboxMapsStyle/Generated
-reporter: "xcode"
-whitelist_rules:
-  - block_based_kvo
-  - class_delegate_protocol
-  - closing_brace
-  - closure_parameter_position
-  - colon
-  - comma
-  - compiler_protocol_init
-  - control_statement
-  - cyclomatic_complexity
-  - deployment_target
-  - discarded_notification_center_observer
-  - discouraged_direct_init
-  - duplicate_enum_cases
-  - duplicate_imports
-  - dynamic_inline
-  - empty_enum_arguments
-  - empty_parameters
-  - empty_parentheses_with_trailing_closure
+  - Sources/MapboxMaps/Style/Generated
+  - Tests/MapboxMapsTests/Style/Generated
+child_config: Tests/.swiftlint.yml
+opt_in_rules:
   - explicit_top_level_acl
-  - file_length
-  - for_where
-  - force_cast
-  - function_parameter_count
-  - generic_type_name
-  - implicit_getter
-  - inert_defer
-  - is_disjoint
-  - large_tuple
-  - leading_whitespace
-  - legacy_cggeometry_functions
-  - legacy_constant
-  - legacy_constructor
-  - legacy_hashing
-  - legacy_nsgeometry_functions
-  - mark
-  - multiple_closures_with_trailing_closure
-  - nesting
-  - no_fallthrough_only
-  - no_space_in_method_call
-  - notification_center_detachment
-  - nsobject_prefer_isequal
-  - opening_brace
-  - operator_whitespace
-  - orphaned_doc_comment
-  - private_over_fileprivate
-  - private_unit_test
-  - protocol_property_accessors_order
-  - reduce_boolean
-  - redundant_discardable_let
-  - redundant_objc_attribute
-  - redundant_optional_initialization
-  - redundant_set_access_control
-  - redundant_string_enum_value
-  - redundant_void_return
-  - return_arrow_whitespace
-  - statement_position
-  - superfluous_disable_command
-  - switch_case_alignment
-  - syntactic_sugar
+disabled_rules:
+  - comment_spacing
+  - computed_accessors_order
+  - force_try
```

Comment:
Why are disabling force_try. This feels like something we'd want to lint on

Reply:
I agree, but it wasn't enabled previously, and enabling it will be a rather large undertaking that I didn't have time to tackle in this PR.

It's on my list of things to attempt in the future.